news/2020-10-14-new-featured-artist-p4koo.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: "New Featured Artist: P4koo"
+date: 2020-10-14 09:00:00 +0000
+---
+
+Rhythm game music veteran **P4koo** joins the lineup as our latest Featured Artist!
+
+![](https://assets.ppy.sh/artists/106/header.jpg)
+
+If you consider yourself any sort of rhythm game conniseur, chances are that you've played one of the many games **P4koo** has featured in - from titles like *EZ2AC (EZ2DJ)*, *O2Jam*, *SOUND VOLTEX*, *Dynamix*, *Mush Dash* and a whole bunch more.
+
+We're excited to bring his unique twist on the classic rhythm game sound (you all know what it is) into the osu! community proper with an ample selection of **33** tracks from across his whole discography, including well-loved tracks already popular here like *Fly High* and *Crystal Illusion*
```

Comment:
missing full stop

``including well-loved tracks already popular here like *Fly High* and *Crystal Illusion*`` --> ``including well-loved tracks already popular here like *Fly High* and *Crystal Illusion*.``